### ISSUE TITLE
cdnjs hosting

### ISSUE THREAD

**Issue by sashberd** [DataTables/Plugins#353]
Hi
We would like to [add ](https://github.com/cdnjs/cdnjs/issues/6646)you library to cdnjs.
Which files we should add?
Thanks

**Comment by DataTables** [comment]
Yup sorry - I've just not got around to updating this repo yet. You could pull in all the current js and css files, but things like the language files need to be changed over from a .lang file to a .js file.

**Comment by sashberd** [comment]
May be you could create a build process with Grunt or Galp and provide us 2 files, regular and minified, in dist folder?
If you dont, please provide exact folder with js and css to take, I do not think We have to take all project with all folders.
Another question - from what version you library you think should be hosted. By default we take all version history, but if the author says we could take from specific one.
Thanks for cooperation

**Comment by DataTables** [comment]
Yes - a build process is what I need to build, but just haven't had the time to do it yet.

Regarding just two files - this repo is a collection of individual plug-ins. It wouldn't be appropriate to just have two files.
